File: app/src/main/java/no/joharei/flixr/mainpage/MainView.java
package no.joharei.flixr.mainpage;

import android.content.Context;

import java.util.List;

import no.joharei.flixr.api.models.Photoset;
import no.joharei.flixr.mainpage.models.Contact;

interface MainView {
    Context getContext();

    void showMyPhotosets(List<Photoset> photosets);

    void showMyContacts(List<Contact> contacts);
}

File: app/src/main/java/no/joharei/flixr/mainpage/MainView.kt
package no.joharei.flixr.mainpage

import android.content.Context
import no.joharei.flixr.mainpage.models.Contact
import no.joharei.flixr.network.models.Photoset

interface MainView {
    fun getViewContext(): Context

    fun showMyPhotosets(photosets: List<Photoset>)

    fun showMyContacts(contacts: List<Contact>)
}
